pkg/puppetdb/client.go

```diff
@@ -18,13 +19,14 @@ type Client struct {
 }
 
 // NewClient access the orchestrator API via TLS
-func NewClient(hostURL, token string, tlsConfig *tls.Config) *Client {
+func NewClient(hostURL, token string, tlsConfig *tls.Config, timeout time.Duration) *Client {
```

Comment:
It would be good to have a comment explaining a sensible default to use. Is it 60 seconds or 10 ?

Reply:
Otherwise, seems sensible.

Reply:
I updated the comment with some more blurb and what we use in ER for the timeout. 

Thanks